src/Sarif.WorkItems/SarifWorkItemModel.cs

```diff
@@ -83,9 +83,14 @@ private string CreateBugFooter()
             if(this.Context.CurrentProvider == FilingClient.SourceControlProvider.AzureDevOps)
             {
                 StringBuilder azureDevOpsFooter = new StringBuilder();
-                azureDevOpsFooter.Append(@"To see result details, please visit the Scans tab of this bug, or the attached SARIF log.");
+                azureDevOpsFooter.Append(@"<br><br>Other viewing options:<br>");
```

Comment:
><br><br> [](start = 43, length = 8)

Sometimes you choose `<br>` and sometimes an extra call to `AppendLine`. Is there a principled reason to choose one over the other on a case-by-case basis? If so it's worth a comment.

Reply:
This is strongly related to the above.  As this is the ADO footer, and it does support HTML, I'll remove the appendLines here.  I do use them elsewhere in more genereal (non-HTML enabled text) where I think they make sense.

---
In reply to: [398187126](https://github.com/microsoft/sarif-sdk/pull/1823#discussion_r398187126) [](ancestors = 398187126)